packages/@expo/config-plugins/src/plugins/mod-compiler.ts

```diff
@@ -103,13 +107,15 @@ function getRawClone({ mods, ...config }: ExportedConfig) {
   return Object.freeze(JSON.parse(JSON.stringify(config)));
 }
 
-const orders: Record<string, string[]> = {
-  ios: [
+const precedences: Record<string, Record<string, number>> = {
+  ios: {
     // dangerous runs first
-    'dangerous',
+    dangerous: 2,
```

Comment:
xit: I’d expect smaller numbers to sort toward the beginning of an array, i.e. like the sort method.